app/models/taxon.rb

```diff
@@ -37,6 +38,20 @@ def redirected?
     publication_state == "unpublished" && !redirect_to.nil?
   end
 
+  def draft_previously_published?
+    previous_state, latest_state = last_two_publication_states
+    previous_state && latest_state == "draft"
+  end
```

Comment:
I think `draft_and_published_editions_exist?` might be a clearer name for this.

Reply:
Sure.